tests/Controller/AdminProductVariantAjaxTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Controller;
+
+use ApiTestCase\JsonApiTestCase;
+use Symfony\Component\BrowserKit\Cookie;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+
+final class AdminProductVariantAjaxTest extends JsonApiTestCase
+{
+    /** @test */
+    public function it_denies_access_to_product_variants_for_not_authenticated_user()
```

Comment:
```suggestion
    public function it_denies_access_to_product_variants_for_not_authenticated_user(): void
```
in the following too